.github/workflows/brakeman.yml

```diff
@@ -20,7 +20,9 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
wondering why we need this defined here?

Reply:
I don't know either.. just removed.. tks Abeer 👍